frontend/webpack.common.js

```diff
@@ -0,0 +1,49 @@
+require('dotenv').config();
+
+const { DefinePlugin } = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const production = process.env.NODE_ENV === 'production';
+
+const webpackConfig = module.exports = {};
+
+webpackConfig.entry = `${__dirname}/src/main.js`;
+
+webpackConfig.output = {
+  filename: `[name].[hash].js`,
+  path: `${__dirname}/build`,
+  publicPath: process.env.CDN_URL,
+};
+
+webpackConfig.plugins = [
+  //Plugin that simplifies creation of HTML files to serve your bundles
+  new HtmlWebpackPlugin({
+    title: 'React App',
+    template: `${__dirname}/src/index.html`,
+  }),
+
+  new DefinePlugin({
+    __API_URL__: JSON.stringify(process.env.API_URL),
```

Comment:
Here is your api url global. Note: The .env file calls it "API_URL" but your webpack is defining it as "__API_URL__" so in your React code you will use the one with dunders, not the one in your .env.